prime-router/src/main/kotlin/config/validation/JsonSchemaValidationService.kt

```diff
@@ -58,8 +58,8 @@ class JsonSchemaValidationServiceImpl : JsonSchemaValidationService {
             val parsedJson = JacksonMapperUtilities.yamlMapper.readTree(inputStream)
             val schemaValidation = configType.jsonSchema.validate(parsedJson)
             schemaErrors = schemaValidation.map { it.message }
-            val parsed = configType.convert(parsedJson)
             if (schemaErrors.isEmpty()) {
+                val parsed = configType.convert(parsedJson)
```

Comment:
moved this one since the vast-majority of the time if a file does not pass json schema validation, it will not parse via jackson either. We can focus on a the schema errors rather than appending a duplicative exception to every error log.